integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/TestActions.java

```diff
@@ -638,10 +638,20 @@ public static void addLabelsToNamespace(String name, Map<String, String> labels)
    * @return true on success, false otherwise
    * @throws ApiException if Kubernetes client API call fails
    */
-  public static String createUniqueNamespace() throws ApiException {
+  public static synchronized String createUniqueNamespace() throws ApiException {
```

Comment:
Do we still need synchronization, since we are making sure that no duplicate name is generated. As we disused, synchronization may block/delay test execution.

Reply:
removed